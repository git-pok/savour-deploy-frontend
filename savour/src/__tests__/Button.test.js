import Buttton from '../Button.js';
import { render } from '@testing-library/react';
import { MemoryRouter } from 'react-router-dom';

test("renders Buttton without crashing", () => {
  render(
    <MemoryRouter>
    <Buttton
      buttonText="HOME"
      link="/home" />
    </MemoryRouter>
  );
});

test("renders snapshot without crashing", () => {
  const { asFragment } = render(
    <MemoryRouter>
      <Buttton
        buttonText="HOME"
        link="/home" />
    </MemoryRouter>
  );
  expect(asFragment()).toMatchSnapshot();
});
