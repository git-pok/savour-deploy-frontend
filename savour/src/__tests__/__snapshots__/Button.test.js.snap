// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders snapshot without crashing 1`] = `
<DocumentFragment>
  <div
    class="Button"
  >
    <a
      exact="true"
      href="/home"
    >
      <button
        class="Button-button"
      >
        HOME
      </button>
    </a>
  </div>
</DocumentFragment>
`;
